posts/flat-file-exports/flat-files-compare.qmd

```diff
@@ -0,0 +1,440 @@
+ $ nCell                                                                   : int  932 932 932 932 932 932
+ $ nCount                                                                  : int  614735 614735 614735 614735 614735 614735
+ $ nCountPerCell                                                           : num  660 660 660 660 660 ...
+ $ nFeaturePerCell                                                         : num  404 404 404 404 404 ...
+ $ propNegativeCellAvg                                                     : num  0.00058 0.00058 0.00058 0.00058 0.00058 ...
+ $ complexityCellAvg                                                       : num  1.54 1.54 1.54 1.54 1.54 ...
+ $ errorCtPerCellEstimate                                                  : num  100 100 100 100 100 ...
+ $ percOfDataFromErrorPerCell                                              : num  0.152 0.152 0.152 0.152 0.152 ...
+ $ qcFlagsFOV                                                              : chr  "Pass" "Pass" "Pass" "Pass" ...
+ $ i.median_negprobes                                                      : num  14 14 14 14 14 14
+ $ i.negprobes_quantile_0.9                                                : num  25.1 25.1 25.1 25.1 25.1 25.1
+ $ i.median_RNA                                                            : int  51 51 51 51 51 51
+ $ i.RNA_quantile_0.9                                                      : num  166 166 166 166 166 166
+```
+
+
+:::
+
+# Transcript coordinates file
+
+
+
+::: {.panel-tabset group="version"}
+
+## Legacy
+
+
+```{r}
+#| eval: false
+
+tx_file_old <- read.csv("path/to/NSCLC/Lung5_Rep1/Lung5_Rep1-Flat_files_and_images/Lung5_Rep1_tx_file.csv", header=TRUE)
+tx_file_old %>% head(10)
+```
+
+
+```
+  fov cell_ID x_global_px y_global_px x_local_px y_local_px  z  target CellComp
+  1       0    6757.402    158836.4   3568.513  3619.7375 11   NEAT1        0
+  1       0    5111.389    156060.2   1922.500   843.5334 11   NEAT1        0
+  1       0    7860.461    157809.3   4671.572  2592.6715 11    CCR2        0
+  1       0    3790.489    155553.9    601.600   337.2168 11 HLA-DRA        0
+  1       0    3290.639    158023.6    101.750  2806.9750 11 HLA-DRA Membrane
+  1       0    7020.160    158656.3   3831.271  3439.6000 11     VHL        0
+  1       0    4252.914    157003.0   1064.025  1786.3376 11    FZD5  Nuclear
+  1       0    5987.309    157572.5   2798.420  2355.8000 11    CD37        0
+  1       0    5586.849    157774.2   2397.960  2557.5599 11   ATG12 Membrane
+```
+
+```{r}
+#| eval: false
+tx_file_old %>% select(CellComp) %>% unique()
+```
+
+```
+  CellComp
+  0
+  Membrane
+  Nuclear
+  Cytoplasm
+
+```
+## AtoMx 1.3.2
+
+
+```{r}
+#| eval: false
+library(plyr)
+library(dplyr)
+tx_file_new <- read.csv("path/to/breast_cancer_example/flatFiles/tmp_tx.csv", header=TRUE)
+tx_file_new %>% tail(10)
+```
+
+
+```
+  fov cell_ID             cell x_local_px y_local_px x_global_px y_global_px        z    target CellComp
```

Comment:
Column names appear to not be aligned with column contents. Is the name 'cell' extra?

Reply:
Good catch. This is resolved. 1a0416d378b30a2f45278643629781e3b497fd92

Reply:
Column names are now all present but 'cell' is not rendering aligned:
![image](https://github.com/Nanostring-Biostats/CosMx-Analysis-Scratch-Space/assets/24260255/bd47059c-6b74-487f-94f0-db0a21662864)
Are there too many tabs or spaces between 'cell_ID' and 'cell'?

Reply:
Manually adjusted STDOUT. Looks like this now: 
<img width="694" alt="image" src="https://github.com/Nanostring-Biostats/CosMx-Analysis-Scratch-Space/assets/5805450/3615d6b2-04c8-4f8c-a797-d9400304d94a">

61b562e09a61e633861003d891b4df50edaa46f5